File: _FACUL/LP/LP_Go/pow.go
package main

import (
	f "fmt"
)

func main() {
	x, a, p := 0, 0, 1
	for x <= 0 {
		f.Print("Digite um numero inteiro positivo: ")
		f.Scanf("%d\n", &x)
	}
	f.Print("Digite o expoente: ")
	f.Scanf("%d\n", &a)
	for a > 0 {
		p *= x
		a--
	}
	f.Println(p)
}

File: _FACUL/LP/LP_C/pow.c
#include <stdio.h>

int main(void) {
  int x, a, p;
  p = 1;
  do{
    printf("\n Digite um numero inteiro positivo: ");
    scanf("%d", &x);
  } while( x <= 0);
  printf("\n Digite o expoente: ");
  scanf("%d", &a);
  while(a-- > 0){
    p *= x;
  }
  printf("\nPow = %d\n", p);
  return 0;
}